source/exploring_catalog_and_datasets/04_getting_involved/getting_involved.rst

```diff
@@ -0,0 +1,29 @@
+Getting involved: sharing, reusing & reacting
+=============================================
+
+As a user of open data, multiple options are available to get more involved into the open data environment. Whether you come accross some datasets that you are particularly interested in, and want to share them with the world. Or whether you have specific skills you believe could be put to use to create something with those published data.
```

Comment:
Not sure about the `user of open data`, it's a bit restrictive to me, it's the same doc for internal sharing or data product (not only open data).

Maybe: `As a data consumer, multiple options are available to get more involved into the open data environment.` ?